appengine-managed-runtime/src/test/java/com/google/apphosting/vmruntime/VmApiProxyDelegateTest.java

```diff
@@ -194,28 +197,30 @@ private void callDelegateWithOneError(
     byte[] result = null;
     final Double timeoutInSeconds = 10.0;
 
-    if (sync) {
-      try {
-        environment.getAttributes().put(VmApiProxyDelegate.API_DEADLINE_KEY, timeoutInSeconds);
-        result =
-            delegate.makeSyncCall(environment, TEST_PACKAGE_NAME, TEST_METHOD_NAME, requestData);
-        fail();
-      } catch (ApiProxyException exception) {
-        assertEquals(expectedException.getClass(), exception.getClass());
-      }
-    } else {
-      try {
-        ApiConfig apiConfig = new ApiConfig();
-        apiConfig.setDeadlineInSeconds(timeoutInSeconds);
-        result =
-            delegate
-                .makeAsyncCall(
-                    environment, TEST_PACKAGE_NAME, TEST_METHOD_NAME, requestData, apiConfig)
-                .get();
-        fail();
-      } catch (ExecutionException exception) {
-        // ExecutionException is expected, and make sure the cause is expected as well.
-        assertEquals(expectedException.getClass(), exception.getCause().getClass());
+    try (SuppressedLogging l=new SuppressedLogging()) {
```

Comment:
I'm not using it because it is making a lot of changes to code I have not touched, so it is producing a different result to all the formatting changes you recently committed.    Mostly wrapping stuff, but I've not had time to investigate.